docs/en/observability/triage-threshold-breaches.asciidoc

```diff
@@ -0,0 +1,44 @@
+[[triage-threshold-breaches]]
+= Triage threshold breaches
+++++
+<titleabbrev>Threshold breaches</titleabbrev>
+++++
+
+Threshold breaches occur when an {observability} data type reaches or exceeds the threshold set in your <<custom-threshold-alert,custom threshold rule>>.
+For example, you might have a custom threshold rule that triggers an alert when the total number of log documents with a log level of `error` reaches 100.
+
+To triage issues quickly, go to the alert details page:
+
+. Go to **{observability}** → **Alerts**.
+. From the Alerts table, click the image:images/icons/boxesHorizontal.svg[More actions] icon next to the alert and select **View alert details**.
+
+The alert details page shows information about the alert, including when the alert was triggered,
+the duration of the alert, and the last status update.
+If there is a "group by" field specified in the rule, the page also includes the source.
+You can follow the links to navigate to the rule definition.
+
+Explore charts on the page to learn more about the threshold breach:
+
+[role="screenshot"]
+image::images/log-threshold-breach.png[Alert details for log threshold breach]
+
+* The page includes a chart for each condition specified in the rule.
+These charts help you understand when the breach occurred and its severity.
+* If your rule is intended to detect log threshold breaches
+(that is, it has a single condition that uses a count aggregation),
+you can run a log rate analysis, assuming you have the required license.
+Running a log rate analysis is useful for detecting significant dips or spikes in the number of logs.
+Notice that you can adjust the baseline and deviation, and then run the analysis again.
+* The page may also include an alerts history chart that shows the last 30 days.
```

Comment:
Shall we also mention that this chart shows the number of triggered alerts per day for the last 30 days?

Reply:
Good idea!